src/main/kotlin/telegram/Handling.kt

```diff
@@ -1179,23 +1182,40 @@ object MafiaHandler {
                 }
             }
             parametrized(nightCommand) {
-                try {
-                    bot.deleteMessage(ChatId.fromId(chatId), long(0))
-                    accounts.update(chatId) {
-                        menuMessageId = -1L
-                    }
-                    deleteGameTimers(bot, game.id)
-                    towns[game.id]?.let { town ->
-                        bot.sendMessage(
+                towns.get(game.id)?.let { town ->
+                    if (town.day == 1 && getHideRolesMode(game)) {
+                        val res = bot.sendMessage(
                             ChatId.fromId(chatId),
-                            "Результат дня:\n${shortLog(town).ifBlank { "Не произошло никаких событий" }}"
+                            "Выключите скрытие ролей в меню настроек"
```

Comment:
Super unnecessary. Just hide `nightCommand` button until they reveal roles and that's it